docs/devenv_cloud.md

```diff
@@ -0,0 +1,142 @@
+* Select the instance type depending on your architecture of choice:
+    * `c6i.2xlarge` for `x86_64`
+    * `c7g.2xlarge` for `aarch64`
+* Select your `Key pair`
+* Select your security group, **only allowing** SSH port 22 inbound traffic
+* Configure storage with the following volumes:
+    * 100 GiB root volume on `gp3` storage type
+    * 10 GiB data volume on `gp3` storage type
+
+Review your selections and lauch the virtual instance. 
+> Unless noted otherwise, all the further configuration should be performed on the cloud virtual instance created in this section.
+
+### Other Cloud Environments
+
+Other public cloud vendors and environments should also be suitable for running the MicroShift development environment, provided the appropriate hardware and operating system configuration is available.
+
+## Development Virtual Machine
+
+Log into the virtual instance running in the cloud using the appropriate login credential (i.e. `ec2-user` for AWS) and a private SSH key.
+
+```bash
+ssh -i ~/.ssh/<private_key> ec2-user@<instance_public_IP>
+```
+
+Create and configure the `microshift` user account, also allowing SSH logins using a password.
+
+```bash
+PASSWORD="<strong password>"
+
+sudo dnf install -y perl
+sudo useradd -m -s /bin/bash -p $(perl -e"print crypt('${PASSWORD}', 'microshift')") microshift
+echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
+
+sudo sed -i 's/PasswordAuthentication.*/PasswordAuthentication yes/g' /etc/ssh/sshd_config
+sudo systemctl reload sshd.service 
+```
+> Choose a strong password and make sure you configure the security group to limit inbound connections on the SSH port 22 **only** from your IP address.
+
+Log into the `microshift` user account and run the following commands to create an LVM volume and group on the data volume of the virtual instance. This disk space will be used by the MicroShift LVMO operator for storing data.
+> Run the `lsblk` command to see the device name of the 10 GiB storage volume allocated during the instance creation.
+
+```bash
+DEVICE=/dev/nvme1n1
+
+sudo dnf install -y lvm2
+sudo pvcreate      ${DEVICE}
+sudo vgcreate rhel ${DEVICE}
+```
+
+Finish by running the `scripts/devenv-builder/configure-vm.sh` script to perform the initial configuration as described in the [Configure Virtual Machine](./devenv_rhel8_auto.md#configure-virtual-machine) document.
+
+## Nested Virtualization
+
+Nested virtualization allows for a greater flexibility, lesser complexity and reuse of the documented procedures when creating RHEL for Edge images in a public cloud. The performance of such a configuration may be reduced, but it is sufficient for the purpose of testing.
+
+### RHEL for Edge Virtual Machine
+
+Log into the development virtual machine running in the cloud and switch to the `microshift` user.
```

Comment:
this looks incoherent with user creation and allowing it to SSH with password ("switch to the user" part)